modules/mobile/docs/schemas/Appointments.yml

```diff
@@ -12,29 +12,36 @@ properties:
     type: object
     additionalProperties: false
     required:
-      - errors
-      - checkinWindow
       - upcomingAppointmentsCount
       - upcomingDaysLimit
+      - pagination
     properties:
       errors:
         type: [array, null]
-      checkinWindow:
-        type: object
-        additionalProperties: false
-        required:
-          - minutesBefore
-          - minutesAfter
-        properties:
-          minutesBefore:
-            type: integer
-            example: 45
-          minutesAfter:
-            type: integer
-            example: 15
       upcomingAppointmentsCount:
         type: number
         description: The number of BOOKED, non-pending appointments in the next upcomingDaysLimit number of days.
       upcomingDaysLimit:
         type: number
         description: The number of days into the future used for calculating upcomingAppointmentsCount. The current value is 7.
+      pagination:
```

Comment:
We should extract pagination into a separate file because it's duplicated across several locations, but I didn't want to bloat this PR.